x-pack/plugins/ml/public/settings/calendar/edit/utils.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+import { ml } from 'plugins/ml/services/ml_api_service';
+import { jobs } from 'plugins/ml/services/ml_api_service/jobs';
```

Comment:
this line is unnecessary as you're already importing the root `ml` object in the line above.
the `jobs` functions can be got at using the namespace `jobs`
e.g. `jobsSummary` is `ml.jobs.jobsSummary()`